README.md

```diff
@@ -32,8 +32,8 @@ FingerprintJS Pro is a professional visitor identification service that processe
 Retrieve an accurate, sticky and stable [FingerprintJS Pro](https://fingerprint.com/) visitor identifier in an Android or an iOS app. This library communicates with the FingerprintJS Pro API and requires an [API key](https://dev.fingerprint.com/docs). 
```

Comment:
Minor, but on the PR to the RN repo there is only Fingerprint Pro mentioning whereas here FingerprintJS Pro. Probably we forgot to change some time ago.